package/MDAnalysis/analysis/align.py

```diff
@@ -504,6 +504,15 @@ def alignto(mobile, reference, select=None, weights=None,
     if subselection is None:
         # mobile_atoms is Universe
         mobile_atoms = mobile.universe.atoms
+        if mobile.universe == reference.universe:
+            # raise a warning when passing atom groups from same Universe
+            wmsg = ("The reference and mobile selections are atom groups"
```

Comment:
Here and other lines, you'll need a whitespace between implicit line returns (i.e. the string should be "... are atom groups " instead of "... are atom groups").

Otherwise what you end up getting is concatenating the final and first word of each line, causing the following message:

"The reference and mobile selections are atom groupsfrom the same universe, the reference selection willbe transformed. If it is not the behavior you intend,try setting the subselection argument to 'all'.`subselection = 'all' is deprecated in 2.1.0 but itwill become the default behavior in version 3.0.0."